packages/sanity/src/core/studio/components/navbar/resources/helper-functions/helpResources.ts

```diff
@@ -0,0 +1,27 @@
+import type {Observable} from 'rxjs'
+import {map} from 'rxjs/operators'
+import {SanityClient} from '@sanity/client'
+import {SANITY_VERSION} from '../../../../../version'
+import {ResourcesResponse} from './types'
+
+/**
+ * @internal Not a stable API yet
+ */
+
+export function getHelpResources(client: SanityClient): Observable<ResourcesResponse> {
+  const request$ = client.observable
+    .request<ResourcesResponse>({
```

Comment:
Not critical since it is currently internal, but it would be safer to enforce an API version here rather than hope the client passed is already using the version we want. Otherwise the `ResourcesResponse` type could theoretically be incorrect.

```suggestion
  const request$ = client
    .withConfig({apiVersion: '1'})
    .observable.request<ResourcesResponse>({
```